osu.Game/Screens/Edit/Screens/Setup/Components/SetupTextBox.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.Edit.Screens.Setup.Components
+{
+    public class SetupTextBox : OsuTextBox
+    {
+        protected override float LeftRightPadding => 15;
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour osuColour)
+        {
+            BorderColour = osuColour.Blue;
+        }
+
+        protected override Drawable GetDrawableCharacter(char c) => new OsuSpriteText { Text = c.ToString(), Colour = BorderColour, TextSize = CalculatedTextSize };
```

Comment:
It doesn't seem like this does anything, remove?

Reply:
It is supposed to change the colour of the text to blue, after loaded in the BDL. If it looks like it does nothing, probably a framework update is required, as https://github.com/ppy/osu-framework/pull/1712 fixed the issue with the text appearing white in all cases.